edit-post/assets/stylesheets/main.scss

```diff
@@ -104,6 +99,11 @@ body.gutenberg-editor-page {
 		font-size: $default-font-size;
 		color: $dark-gray-500;
 	}
+
+	svg.dashicon {
```

Comment:
If these styles are targeted at Dashicon, seems sensible that they be added to a `components/dashicon/style.scss`, also because they don't seem specific to Gutenberg.

Reply:
As far as I know, `components/dashicon/index.js` is a copy from the Dashicons repo, how to import a new `components/dashicon/style.scss` in this file if then it gets overridden with any update from the Dashicons repo?

Reply:
It's a tough one. I think long-term this approach of copy-paste from the Dashicons repository is not very sustainable. Ideally we'd just `import` from a Dashicons module. Where that repository lives, I'm not sure (it could even stay where it's at, or be merged to here). Then, we'd change `components/dashicon/index.js` to...

```js
/**
 * WordPress Dependencies
 */
import Dashicon from '@wordpress/dashicon';

/**
 * Internal dependencies
 */
import './style.scss';

export default Dashicon;
```

It's an open question though whether those styles should be bundled with the shared component itself. Alternatively, we could assign the styles as an inline `style` attribute, though of course these are much harder to override in their high specificity.

It's possible (though not implemented in Gutenberg anywhere) to implement the styles as a stylesheet from the Dashicon module and import into the project as well.

Reply:
I understand all the concerns 🙁  However, this issue is specifically about changing the CSS selector used for this rule. The placement of the rule is unrelated. Can we split the two issues and move on?

Reply:
In resolving one issue, you've introduced two others which did not exist previously (styles applied to a component outside its stylesheet, ad hoc inheriting styles from a component elsewhere in the codebase). Arguably the style placement code conventions are less important than the user-impacting ones fixed here, but all the same I'm not inclined to give blessing to the accumulation of technical debt.

Reply:
@aduth: the styles were already in this file.  I've just added `.dashicon` 🙂 I haven't moved anything nor introduce a new issue. The issue you're pointing out is a pre-existing issue and unrelated.

Reply:
>I've just added `.dashicon` 🙂

This _is_ the issue.

Reply:
It really doesn't change the original issue. The previous CSS selector was _already_ styling the dashicon SVGs.